pkg/events/authorization.go

```diff
@@ -24,7 +24,7 @@ func newAction(as string, scope security.Scope) security.Action {
 // Actions used for authorization module
 var (
 	EVGetActions       = newAction("EV_GetActions", security.ScopeGlobal)
-	EVGetEvents        = newAction("EV_GetEvents", security.ScopeGlobal)
+	EVGetEvents        = newAction("EV_GetEvents", security.ScopeRealm)
 	EVGetEventsSummary = newAction("EV_GetEventsSummary", security.ScopeGlobal)
```

Comment:
JIRA ticket was also suggesting to move EV_GetEventsSummary to ScoreRealm... not necessary?

Reply:
fixed